invenio_app_ils/config.py

```diff
@@ -665,6 +669,37 @@ def _(x):
         update_permission_factory_imp=deny_all,
         delete_permission_factory_imp=deny_all,
     ),
+    fssid=dict(
+        pid_type=FRONT_SITE_SEARCH_PID_TYPE,
+        pid_minter=FRONT_SITE_SEARCH_PID_MINTER,
+        pid_fetcher=FRONT_SITE_SEARCH_PID_FETCHER,
+        search_class=FrontSiteSearch,
+        record_serializers={
+            "application/json": (
+                "invenio_app_ils.records.serializers:json_v1_response"
+            )
+        },
+        search_serializers={
+            "application/json": (
+                "invenio_app_ils.records.serializers:json_v1_search"
+            ),
+            "text/csv": ("invenio_app_ils.records.serializers:csv_v1_search"),
+        },
+        search_serializers_aliases={
+            "csv": "text/csv",
+            "json": "application/json",
+        },
+        item_route="/frontsite-search/<pid(fssid):pid_value>",
```

Comment:
I would suggest to change `frontsite-search`: what about if we do `/records`?

Reply:
I'm not happy with the "front site search" naming here either. It would be better if it was more generic and indicative of what it searches. `/records` is also a bit misleading too since it only searches documents and series and not all records.

How about `/literature`? We discussed using that for the front site routes when we planned the periodicals feature and would kind of make sense in this context. What do you think @kprzerwa @zzacharo @topless?

Reply:
We discussed this on the standup and decided to call it literature instead of "front site search".